Formula/uriparser.rb

```diff
@@ -11,14 +19,6 @@ class Uriparser < Formula
     sha256 "0657e76e94b481bc0b859ba68b8e31d460dce44e7ec3fcc573cb5bfd6bb89839" => :sierra
   end
 
-  head do
-    url "https://github.com/uriparser/uriparser.git"
-
-    depends_on "autoconf" => :build
-    depends_on "automake" => :build
-    depends_on "libtool" => :build
-  end
-
   depends_on "cmake" => :build
   depends_on "pkg-config" => :build
```

Comment:
Do you still need pkg-config if using CMake?

Reply:
No, and I don't need automake/libtool.

Reply:
Building with `--HEAD` no longer requires automake & libtool? If so that's great. I'm skeptical though.

If you don't need pkg-config anymore, can you then please remove the dependency on pkg-config?